Dockerfile

```diff
@@ -31,8 +31,18 @@ RUN curl -o nmc-aarch64 -L https://github.com/suse-edge/nm-configurator/releases
     chmod +x nmc-x86_64 && \
     cp nmc-$(uname -m) /usr/local/bin/nmc
 
+
+RUN curl -o hauler-amd64.tar -L https://github.com/rancherfederal/hauler/releases/download/v0.4.2/hauler_0.4.2_linux_amd64.tar.gz && \
```

Comment:
This means that we will also need to build hauler in OBS/IBS if we want to get EIB into the suse registry.